src/pages/DisbursementDetails.tsx

```diff
@@ -194,6 +194,20 @@ export const DisbursementDetails = () => {
                 </CopyWithIcon>
               </div>
             </div>
+
+            <div className="StatCards__card__item">
+              <label className="StatCards__card__item__label">Created by</label>
+              <div className="StatCards__card__item__value">
+                {disbursementDetails.details.createdBy}
```

Comment:
We might need to add a fallback here to make sure UI doesn't display `undefined` (same below).
```suggestion
                {disbursementDetails.details?.createdBy ?? ""}
```